project.php

```diff
@@ -317,13 +317,14 @@ function decide_blurbs()
 
         // Has the user saved a page of this project since the project comments were
         // last changed? If not, it's unlikely they've seen the revised comments.
-        $res = mysqli_query(DPDatabase::get_connection(), "
+        $sql = "
             SELECT {$round->time_column_name}
             FROM $projectid
-            WHERE state='{$round->page_save_state}' AND {$round->user_column_name}='$pguser'
+            WHERE state='{$round->page_save_state}' AND {$round->user_column_name}='" . DPDatabase::escape($pguser) . "'
```

Comment:
no sprintf?

Reply:
It's awkward to use` sprintf` in this case because most of the interpolated variables are field names and the style guide [says](https://www.pgdp.net/wiki/DP_Code_Best_Practices#Column_names_as_variables) to keep those inline rather than using `%s`. In other places with similar issues, Casey suggested using concatenation like this.

Reply:
Yes, this is less bad than the alternatives.